podvm/Dockerfile.podvm_builder

```diff
@@ -29,6 +29,8 @@ RUN apt-get update -y && \
     echo "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | tee -a /etc/apt/sources.list && \
     apt-get update && apt-get install --no-install-recommends -y packer && \
     apt-get clean && \
+    wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 && \
+    chmod a+x /usr/local/bin/yq && \
```

Comment:
We should pin a version instead of pulling in latest.

Also, there is a very nice experimental syntax in docker, that can be used to add hash pinned dependencies:
https://github.com/confidential-containers/cloud-api-adaptor/blob/27af578f30e6d4f5151f6ee807e33e794e2edfde/hack/Dockerfile.golang#L1
 https://github.com/confidential-containers/cloud-api-adaptor/blob/27af578f30e6d4f5151f6ee807e33e794e2edfde/hack/Dockerfile.golang#L13

We can replace every wget in a dockerfile with this (must not necessarily happen in this pr).